cmd/snap-bootstrap/cmd_initramfs_mounts_test.go

```diff
@@ -6532,7 +6533,7 @@ func (s *initramfsMountsSuite) TestInitramfsMountsTryRecoveryHealthCheckFails(c
 	c.Assert(os.MkdirAll(filepath.Dir(mockedState), 0750), IsNil)
 	c.Assert(ioutil.WriteFile(mockedState, []byte(mockStateContent), 0640), IsNil)
 
-	restore = main.MockTryRecoverySystemHealthCheck(func() error {
+	restore = main.MockTryRecoverySystemHealthCheck(func(gadget.Model) error {
```

Comment:
we have tests where isClassic is true, I'm just surprised we can't check anything about the changes in any test

Reply:
I looked a bit and pushed one change to make things a bit more explicit (in `makeRunSnapSystemdMount` which is used to test the classic mounts). The other one tests TestInitramfsMountsTryRecoveryHappy on classic which ensures that the different paths are used. Let me know if I misunderstood or there is more that should be tested here !